### ISSUE TITLE
在 SecretFlow 中实现个性化联邦策略 FedBN

### ISSUE THREAD

**Issue by Candicepan** [secretflow/secretflow#996]
> 此 ISSUE 为 [隐语开源共建计划（SecretFlow Open Source Contribution Plan，简称 SF OSCP）第三期](https://github.com/orgs/secretflow/discussions/1007)任务 ISSUE，欢迎社区开发者参与共建～
> 若有感兴趣想要认领的任务，但还未报名，辛苦先完成[报名](https://www.wjx.top/vm/QKio0dq.aspx# )进行哈～

## 任务介绍
- **任务名称**：在 SecretFlow 中实现个性化联邦策略 FedBN
- **技术方向**：FL/strategy
- **任务难度**：进阶🌟🌟

## 详细要求
- 安全性：尽量少 reveal
- 功能性：在 SecretFlow 中实现论文[FedBN: Federated Learing on Non-IID Features VIA Local Batch Normalization](https://arxiv.org/pdf/2102.07623.pdf)的算法。
- 正确性：至少复现论文中的一个实验验证实现的正确性
- 代码规范：Python 代码需要使用 black+isort 进行格式化（流水线包含代码规范检查卡点）
- 框架后端：实现 PyTorch 版本。若在完成 PyTorch 版本实现的基础上同步进行 TensorFlow 版本实现将额外累积一颗🌟。
- 提交说明：关联该 isuue 并提交代码至 https://github.com/secretflow/secretflow/blob/main/secretflow/ml/nn/fl/backend/torch/strategy/

## 能力要求
- 了解基本 git 操作
- 熟悉 深度学习、联邦学习
- 对 SecretFlow 有一定了解

## 操作说明
- 范例：[PyTorch 后端](https://github.com/secretflow/secretflow/blob/main/secretflow/ml/nn/fl/backend/torch/strategy/fed_prox.py) 和 [TensorFlow 后端](https://github.com/secretflow/secretflow/blob/main/secretflow/ml/nn/fl/backend/torch/strategy/fed_prox.py)

**Comment by Starlight039** [comment]
Starlight039 Give it to me
